packages/react-search-ui/src/containers/__tests__/Facet.test.js

```diff
@@ -269,3 +269,68 @@ it("passes className through to the view", () => {
   );
   expect(viewProps.className).toEqual(className);
 });
+
+describe("search facets", () => {
+  let wrapper;
+  const field = 'field1';
+
+  function subject(additionalProps = {}) {
+    return shallow(
+      <FacetContainer
+        {...{
+          ...params,
+          field,
+          facets: {
+            field1: [
+              {
+                field,
+                data: [
+                  { count: 20, value: "Virat" },
+                  { count: 10, value: "dhoni" },
+                  { count: 9, value: "LÒpez" },
+                  { count: 8, value: "bumŗÄh" },
+                ],
+                type: "value"
+              }
+            ]
+          },
+          isFilterable: true,
+          ...additionalProps
+        }}
+      />
+    );
+  }
+
+  beforeAll(() => {
+    wrapper = subject();
+  });
+
+  it("should have a search input", () => {
+    expect(wrapper.find(View).prop("showSearch")).toEqual(true);
+  });
+
+  it("should use the field name as a search input placeholder", () => {
+    expect(wrapper.find(View).prop("searchPlaceholder")).toBe(`Search ${field}`);
+  });
+
+  it("should render matched Facet options", () => {
```

Comment:
Would you mind writing another test or two explicitly calling out the different matching behaviors? Uppercase vs lowercase, and accent folding? And any other useful cases you can think of?